data/heb0001/heb010/__cts__.xml

```diff
@@ -0,0 +1,13 @@
+<ti:work xmlns:cpt="http://capitains.github.io/xmlns" xmlns:dc="http://purl.org/dc/elements/1.1" xmlns:dct="http://purl.org/dc/terms/" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:hebrewlit:heb0001" xml:lang="eng" urn="urn:cts:hebrewlit:heb0001.heb010">
+  <cpt:structured-metadata>
+    <dc:version></dc:version>
+  </cpt:structured-metadata>
```

Comment:
Sorry, I meant the `<cpt:structured-metadata>` node above this comment, i.e., the one that has the empty `<dc:version>` element. And add the `<cpt:structured-metadata>` node that surrounds the `<dct:hasVersion>` node.